### PR TITLE
docs(graphql): add docstrings to mutations.py

### PR DESCRIPTION
**Pull request by rexledesma**
### Summary & Motivation
As the title.

Resolves https://github.com/dagster-io/dagster/issues/4097.

### How I Tested These Changes
```
tox -e pylint
make graphql
```

### CODE REVIEW

**Comment by rexledesma** [comment]
Current dependencies on/for this PR:
* master
  * **PR #7669** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/7669" target="_blank">Graphite</a> 
    * **PR #7691** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/7691" target="_blank">Graphite</a>  (this PR)

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/dagster-io/dagster/7691?utm_source=stack-comment).

File: python_modules/dagster-graphql/dagster_graphql/schema/roots/mutation.py [hunk 1]
```diff
@@ -133,13 +132,17 @@ def create_execution_metadata(graphql_execution_metadata):
 
 
 class GrapheneDeletePipelineRunSuccess(graphene.ObjectType):
+    """Output indicating that a historical run was successfully deleted."""
```
Comment (gibsondan) [inline python_modules/dagster-graphql/dagster_graphql/schema/roots/mutation.py:135]: drop historical?

File: python_modules/dagster-graphql/dagster_graphql/schema/roots/mutation.py [hunk 2]
```diff
@@ -167,13 +172,17 @@ def mutate(self, graphene_info, **kwargs):
 
 
 class GrapheneTerminatePipelineExecutionSuccess(graphene.Interface):
+    """Interface indicating that a run was successfully terminated/canceled."""
```
Comment (gibsondan) [inline python_modules/dagster-graphql/dagster_graphql/schema/roots/mutation.py:175]: Interface indicating that a run was terminated.

File: python_modules/dagster-graphql/dagster_graphql/schema/roots/mutation.py [hunk 3]
```diff
@@ -167,13 +172,17 @@ def mutate(self, graphene_info, **kwargs):
...
+
     run = graphene.Field(graphene.NonNull(GrapheneRun))
 
     class Meta:
         name = "TerminatePipelineExecutionSuccess"
 
 
 class GrapheneTerminateRunSuccess(graphene.ObjectType):
+    """Output indicating that a run was successfully terminated/canceled."""
```
Comment (gibsondan) [inline python_modules/dagster-graphql/dagster_graphql/schema/roots/mutation.py:184]: drop canceled?

File: python_modules/dagster-graphql/dagster_graphql/schema/roots/mutation.py [hunk 4]
```diff
@@ -182,6 +191,8 @@ class Meta:
 
 
 class GrapheneTerminatePipelineExecutionFailure(graphene.Interface):
+    """Interface indicating that a run failed to terminate/cancel."""
```
Comment (gibsondan) [inline python_modules/dagster-graphql/dagster_graphql/schema/roots/mutation.py:194]: drop cancel?

File: python_modules/dagster-graphql/dagster_graphql/schema/roots/mutation.py [hunk 5]
```diff
@@ -332,11 +351,13 @@ def mutate(self, graphene_info, **kwargs):
...
 class GrapheneTerminateRunPolicy(graphene.Enum):
+    """The type of termination/cancellation policy to use for a run."""
+
     # Default behavior: Only mark as canceled if the termination is successful, and after all
-    # resources peforming the execution have been shut down.
+    # resources performing the execution have been shut down.
     SAFE_TERMINATE = "SAFE_TERMINATE"
 
-    # Immediately mark the pipelie as canceled, whether or not the termination was successful.
+    # Immediately mark the pipeline as canceled, whether or not the termination was successful.
```
Comment (gibsondan) [inline python_modules/dagster-graphql/dagster_graphql/schema/roots/mutation.py:360]: s/pipeline/run

File: python_modules/dagster-graphql/dagster_graphql/schema/roots/mutation.py [hunk 6]
```diff
@@ -377,13 +402,17 @@ class Meta:
...
+    """Output indicating that a code location was successfully shut down."""
+
     repositoryLocationName = graphene.NonNull(graphene.String)
 
     class Meta:
         name = "ShutdownRepositoryLocationSuccess"
 
 
 class GrapheneShutdownRepositoryLocationMutationResult(graphene.Union):
+    """The output from shutting down a code location."""
```
Comment (gibsondan) [inline python_modules/dagster-graphql/dagster_graphql/schema/roots/mutation.py:414]: shutting down a code location server.

**Review by gibsondan — APPROVED** [review]

**Pull request merged**